HazelcastCloudTests/csharphazelcastcloudtests/Remote/Resources/alternative.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need this file?

Reply:
We don't anymore. We have removed these dependencies on the origin but here. The remote folder is copied into testing helpers of origin repo. Here is outdated. That's the reason why  it's here. Now, removed and aligned with origin repo.